### PR TITLE
First task- testing the sumator for strings

### PR DESCRIPTION
**Pull request by KrasimirRaikov**
1.testing if it returns correct result with correct input
2.testing if it throws exception with null parameter input
3.testing if it throws exception with input parameter symbol different than integer

### CODE REVIEW

File: src/sumator/TestSumator.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package sumator;
+
+import junit.framework.TestCase;
+/**
+ * @author raikov.krasimir@gmail.com (Krasimir Raikov)
+ */
+public class TestSumator  extends TestCase{
```
Comment (mgenov) [inline src/sumator/TestSumator.java]: As we are using java 1.7 (there is and 1.8) junit 4 need to be used, cause it provides better experience. 

Name of the class need to be changed to SumatorTest, cause it's a pattern which is used anywhere. As example you can take a look at: https://github.com/google/guava/tree/master/guava-tests/test/com/google/common/escape

Comment (mgenov) [inline src/sumator/TestSumator.java]: Guava is using old junit 3, cause this is really old project and will require a lot of rewriting.

File: src/sumator/TestSumator.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+  public void testSum(){
+    int i= sumator.sum("1", "2");
+    assertEquals(3, i);
+  }
+
+  public void testSumException(){
+    try{
+      sumator.sum("", "2");
+      fail();
```
Comment (mgenov) [inline src/sumator/TestSumator.java]: This will give better report: 
fail("expected exception to be thrown when sum '' and '2', but it didn't happen");

**Comment by KrasimirRaikov** [comment]
sorry, i'll fix it

**Comment by KrasimirRaikov** [comment]
is it ok now?

**Comment by mgenov** [comment]
Still junit 3 

Junit 4 is using annotations

**Comment by KrasimirRaikov** [comment]
I thing I did it, have I missed anything?

File: src/sumator/SumatorTest.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    sumator.sum("", "2");
+  }
+
+  @Test(expected = IllegalArgumentException.class)
+  public void SumSymbolException() {
+    sumator.sum("s", "4");
+
+  }
+
+  @After
```
Comment (mgenov) [inline src/sumator/SumatorTest.java]: You don't have to put this method below others. Mostly Before and After are placed at top of the test, cause they are representing what will happen before test and after the test. If you have long list of tests and you are wondering what happens after test is executed, that you have to scroll.

File: src/sumator/SumatorTest.java [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+public class SumatorTest {
+  private Sumator sumator;
+
+  @Before
+  public void setUp() {
+    sumator = new Sumator();
+  }
+
+  @Test
+  public void Sum() {
```
Comment (mgenov) [inline src/sumator/SumatorTest.java]: Name of the methods in java are starting with lower case characters. I don't know why you changed this.

**Comment by mgenov** [comment]
LGTM ! 

You are free to merge your changes after method naming is fixed.

**Pull request merged**